dm/metrics/alertmanager/dm_worker.rules.yml

```diff
@@ -24,11 +24,11 @@ groups:
       summary: DM remain storage of relay log
 
   - alert: DM_relay_process_exits_with_error
-    expr: changes(dm_relay_exit_with_error_count[1m]) > 0
+    expr: changes(dm_relay_exit_with_error_count{resumable_err="false"}[1m]) > 0 or on(instance, job) increase(dm_relay_exit_with_error_count{resumable_err="true"}[1m]) > 3
```

Comment:
change `expr` to `increase(dm_relay_exit_with_error_count{resumable_err="true"}[2m]) > 3` in case the resumeable error is a `connection timeout` which takes 30s, @lance6716 ptal